Gemfile

```diff
@@ -13,3 +13,4 @@ gem 'oga'
 gem 'ox'
 gem "unitsml"
 gem "debug"
+gem "mml", git: "https://github.com/plurimath/mml", branch: "features/mathml_parsing"
```

Comment:
We need to remove the git address, I'm sure you will...

Reply:
> We need to remove the git address

@ronaldtse, Yes, I’ll do that but we’ll need to release the `mml` gem’s version before that.
Can you review/approve the plurimath/mml#2 first, so I can merge and release the version first?